src/api/libcellml/annotator.h

```diff
@@ -652,6 +652,27 @@ class LIBCELLML_EXPORT Annotator: public Logger
      */
     std::vector<std::string> duplicateIds();
 
+    /**
+     * @brief Assign an automatically generated, unique identifier to the given @p item.
+     *
+     * Assign an automatically generated, unique identifier to the given @p item.
+     * This method will return the new identifier that has been assigned, or an empty string
+     * if the operation failed.
+     *
+     * An identifier will not be assigned if:
+     *  - no model has been stored in this annotator;
+     *  - the given @p item is not a member of the stored model;
+     *  - the given @p item is not relevant to the given @p type; or
+     *  - the given @p item is @c nullptr.
+     *
+     * @see errorCount() if and/or error(size_t) for any issues that may have been raised.
```

Comment:
"error(size_t)"? Hmm... not sure we want that kind of description. People should have to know what `size_t` is about.

Reply:
I think this should be `@see error(size_t)` so that doxygen can find the method.